transport/grpc/handler.go

```diff
@@ -60,67 +59,28 @@ func (h *handler) handle(srv interface{}, serverStream grpc.ServerStream) error
 		return errInvalidGRPCStream
 	}
 
-	var requestData []byte
-	if err := serverStream.RecvMsg(&requestData); err != nil {
+	streamMethod := stream.Method()
+	transportRequest, err := h.getBasicTransportRequest(ctx, streamMethod)
```

Comment:
Basic is one of the weasel words. I think we can live without it, without much ambiguity. If we need specificity, we can have getStreamingTransportRequest, on the other side of the naming problem.